tools/check_pod_errors

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+<< 'heredoc_pod_error_rule'
+The script checks that no POD error are present in any changed file.
+heredoc_pod_error_rule
+
+libs_files=$(git ls-files lib/ | grep '.pm$' | xargs echo)
+tmpfile=$(mktemp)
+tmperrorfile=$(mktemp)
+
+success=1
+if test -n "$libs_files"; then
+    for libfile in $libs_files; do
+        perldoc -T -D "${libfile}" 2>/dev/null 1>"$tmpfile"
+        grep -n "POD ERRORS" "$tmpfile" || continue
```

Comment:
I guess this can be a `grep -q` instead of a `grep -n` as nothing is done with the command output just yet.